.github/workflows/update-version-label.yml

```diff
@@ -0,0 +1,29 @@
+name: PR merge to "Next Release" label
+
+on:
+  pull_request:
+    types:
+      - closed
```

Comment:
I'm not sure this will do what you intend. 

If you intend to tag a series of issues in Linear when they get moved along from develop to staging, I think you want to:

- Watch for a PR to be merged from develop to staging
- Parse through each merge commit in the PR
- Find the associated PRs for each merge commit
- Scrape the comments from each merged PR to develop for `LAND-*` IDs
- Assign the label in Linear for each